src/EditorFeatures/Next/Options/EditorConfigDocumentOptionsProvider.DocumentOptions.cs

```diff
@@ -10,37 +12,42 @@ internal sealed partial class EditorConfigDocumentOptionsProvider : IDocumentOpt
         private class DocumentOptions : IDocumentOptions
         {
             private ICodingConventionsSnapshot _codingConventionSnapshot;
+            private readonly IErrorLoggerService _errorLogger;
 
-            public DocumentOptions(ICodingConventionsSnapshot codingConventionSnapshot)
+            public DocumentOptions(ICodingConventionsSnapshot codingConventionSnapshot, IErrorLoggerService errorLogger)
             {
                 _codingConventionSnapshot = codingConventionSnapshot;
+                _errorLogger = errorLogger;
             }
 
             public bool TryGetDocumentOption(Document document, OptionKey option, out object value)
             {
                 var editorConfigPersistence = option.Option.StorageLocations.OfType<EditorConfigStorageLocation>().SingleOrDefault();
+                var namingEditorConfigPersistence = option.Option.StorageLocations.OfType<NamingEditorConfigStorageLocation>().SingleOrDefault();
 
-                if (editorConfigPersistence == null)
+                if (editorConfigPersistence == null && namingEditorConfigPersistence == null)
                 {
                     value = null;
                     return false;
                 }
 
-                if (_codingConventionSnapshot.TryGetConventionValue(editorConfigPersistence.KeyName, out value))
+                var allRawConventions = _codingConventionSnapshot.AllRawConventions;
+                try
                 {
-                    try
+                    if (namingEditorConfigPersistence != null)
                     {
-                        value = editorConfigPersistence.ParseValue(value.ToString(), option.Option.Type);
```

Comment:
I think you've removed the only call to the public ParseValue, and you could delete it.